todo/urls.py

```diff
@@ -0,0 +1,25 @@
+from django.urls import path
```

Comment:
Add blank line between different import groups

use absolute import instead of relative